assets/queries/k8s/allways_pull_images_admission_control_plugin_not_set/query.rego

```diff
@@ -0,0 +1,40 @@
+package Cx
+
+import data.generic.common as common_lib
+import data.generic.k8s as k8sLib
+
+CxPolicy[result] {
+	resource := input.document[i]
+	metadata := resource.metadata
+	specInfo := k8sLib.getSpecInfo(resource)
+	types := {"initContainers", "containers"}
+	container := specInfo.spec[types[x]][j]
+	common_lib.inArray(container.command, "kube-apiserver")
+	not hasFlagWithValue(container, "--enable-admission-plugins", "AlwaysPullImages")
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("metadata.name={{%s}}.%s.%s.name={{%s}}.command", [metadata.name, specInfo.path, types[x], container.name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": "--enable-admission-plugins flag contains 'AlwaysPullImages' plugin",
+		"keyActualValue": "--enable-admission-plugins flag does not contain 'AlwaysPullImages' plugin",
+		"searchLine": common_lib.build_search_line(split(specInfo.path, "."), [types[x], j, "command"]),
+	}
+}
+
+hasFlagWithValue(container, flag, value) {
+	command := container.command
+	startswith(command[a], flag)
+	values := split(command[a], "=")[1]
+	hasValue(values, value)
+} else {
+	args := container.args
+	startswith(args[a], flag)
+	values := split(args[a], "=")[1]
+	hasValue(values, value)
+}
+
+hasValue(values, value) {
+	splittedValues := split(values, ",")
+	splittedValues[_] == value
+}
```

Comment:
As already suggested by @JoaoDanielRufino, you can add this code to the k8s library.

If you add it, please update `kics\assets\queries\k8s\authorization_mode_set_to_always_allow`